detoks/src/main/java/nl/tudelft/trustchain/detoks/DeToksCommunity.kt

```diff
@@ -99,11 +123,79 @@ class DeToksCommunity(private val context: Context) : Community() {
         }
     }
 
+    override fun walkTo(address: IPv4Address) {
+        super.walkTo(address)
+
+        discoveredAddressesContacted[address] = Date()
+    }
+
+    override fun onIntroductionResponse(peer: Peer, payload: IntroductionResponsePayload) {
+        super.onIntroductionResponse(peer, payload)
+
+        if (peer.address in DEFAULT_ADDRESSES) {
+            lastTrackerResponses[peer.address] = Date()
+        }
+    }
+
+    fun broadcastLike(vid: String, torrent: String, creator: String) {
+        Log.d("DeToks", "Liking: $vid")
+
+        // TODO: fix this
```

Comment:
Is this fixed? Also don't submit commented out code (talking about the next block of commented out code)